README.md

```diff
@@ -40,6 +40,7 @@ browser
   }, stack)
   .test.exists('.forum-topiclines')
   .run(function(err) {
+    browser.close();
```

Comment:
IMHO this example is not perfect to recommend:

1. It's more simple to call `.run(true, callback)` in your case. 
2. `.close()` will work too, but designed for standalone calls. For example, in `after(...)` section of tests.